CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 - Do not throw exception when inspecting empty switch statements ([#2995](https://github.com/spotbugs/spotbugs/issues/2995))
 - Adjust priority since relaxed mode reports even `IGNORED_PRIORITY` ([#2994](https://github.com/spotbugs/spotbugs/issues/2994))
 - Fix duplicated log4j2 jar in distribution ([#3001](https://github.com/spotbugs/spotbugs/issues/3001))
+- System property `findbugs.refcomp.reportAll` is now being used. For some new conditions, it will emit an experimental warning.
```

Comment:
Can you please merge or rebase the master branch to get the latest version of the change log and move the new entry to the "unreleased" part?
Otherwise, LGTM!

Reply:
updated the changelog at [43b9f1c](https://github.com/spotbugs/spotbugs/pull/2988/commits/43b9f1c6f255f28e96e860e098994b7f04337031)